packages/gatsby-theme/src/blocks/ShareBlock/ShareBlock.stories.mdx

```diff
@@ -1,38 +1,41 @@
-import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks"
+import { Meta, Story, ArgsTable, Canvas } from "@storybook/addon-docs/blocks"
 import { useArgs } from "@storybook/client-api"
 import _ from "lodash"
 import ShareBlock from "."
 
 export const data = [
   {
     social: "twitter",
-    title: "Octahedroid About Page",
-    to: "https://octahedroid.com/about/",
-    handler: "Octonauts",
-    hashtags: ["Octonautas", "Octahedroid"],
+    title: "Twitter",
+    to: "https://twitter.com/",
+    handler: "a handler",
+    hashtags: ["#firsthashtag", "#secondhashtag"],
     disabled: false,
   },
   {
     social: "linkedin",
+    title: "Linkedin",
+    to: "https://www.linkedin.com/",
     source: "The source",
     summary: "A summary",
     disabled: false,
   },
   {
     social: "whatsapp",
+    title: "Whatsapp",
     separator: "/",
     disabled: false,
```

Comment:
There is a PropType error for a missing `to` prop. We should check if Whatsapp accepts the prop. If it does, we should add it to the story, but if it does not, we should update the PropType declaration for the component.